bsp/boards/iot-lab_M3/board.c

```diff
@@ -19,6 +19,11 @@
 #include "gpio.h"
 #include "cryptoengine.h"
 
+
+//=========================== variables ============================================

```

Comment:
these headers should be always the same width:
- `//==` at the beginning
- `==` until column 30, then description
- the `==`  until column 80
- empty afterwards